src/tests/ftest/pool/BadExclude.py

```diff
@@ -28,12 +28,9 @@
 import sys
 import json
 import ctypes
-from avocado import Test, main
+from apricot import Test
+from avocado import main
```

Comment:
(pylint-import-error) Unable to import 'avocado'

(pylint-unused-import) Unused main imported from avocado